src/main.c

```diff
@@ -0,0 +1,140 @@
+#include "global.h"
+#include "main.h"
+
+extern void sub_08001B20(void);
+extern void sub_08001B14(void);
+extern void sub_08001A1C(void);
+extern void sub_08001AEC(int);
+extern void sub_08001C30(void);
+
+extern int gUnk_03002968;
+
+extern int* gUnk_03002970[5];
+extern int gUnk_08BDA40C;
+extern int gUnk_098C8D9C;
+extern int gUnk_09718FFC;
+extern int gUnk_094D446C;
+extern int gUnk_09E2261C;
```

Comment:
```suggestion
extern const int gUnk_08BDA40C[];
extern const int gUnk_098C8D9C[];
extern const int gUnk_09718FFC[];
extern const int gUnk_094D446C[];
extern const int gUnk_09E2261C[];
```
Data in ROM is always const. And clearly they are arrays.